core/utils.py

```diff
@@ -20,5 +20,29 @@ def wrapper(self, request, *args, **kwargs):
             return JsonResponse({'ERROR' : 'INVALID_USER'}, status=400)
             
         return func(self, request, *args, **kwargs)
+    return wrapper
+
+def SigninCheckDecorator(func):
+    def wrapper(self, request, *args, **kwargs):
+        try:
+            token        = request.headers.get('Authorization', None)
+
+            if not token:
+                request.user = 0
+                return func(self, request, *args, **kwargs)
```

Comment:
0 보다는 None이 좀 더 명확해 보입니다.
```suggestion
            token        = request.headers.get('Authorization')

            if not token:
                request.user = None
                return func(self, request, *args, **kwargs)
```

Reply:
할당 값을 None으로 수정하였고 테스트도 완료하였습니다.